### ISSUE TITLE
Open Discussion

### ISSUE THREAD

**Issue by Bili-Sakura** [Aiolus-X/Norm-INR#3]
Hi, I am extremely interested your work 'Batch Normalization Alleviates the Spectral Bias in Coordinate Networks'.

To my understanding, you find that **batch normaliztion layers** are important to model performance across 2D Image, 2D CT, 3D MRI and etc. tasks. Interestingly, a recent [paper](http://arxiv.org/abs/2507.07101) conduct experiments on language modeling which states `batch_size=1` is more robust.

So I am currently working with time-series data regression task, where $`S`$ is data dimension and $`L`% is time dimension. 

```math
 X \in R^{S \times L},  Y \in R^{S}
```

and we model $`f_\theta(X)=Y`$.  In some existing literature, people adopt 2D-CNN/ResNet that consider it as a image like data. 

I wonder do you have any idea that batch norm would work for our task.

---

```bibtex
@misc{marekSmallBatchSize2025,
  title = {Small {{Batch Size Training}} for {{Language Models}}: {{When Vanilla SGD Works}}, and {{Why Gradient Accumulation Is Wasteful}}},
  shorttitle = {Small {{Batch Size Training}} for {{Language Models}}},
  author = {Marek, Martin and Lotfi, Sanae and Somasundaram, Aditya and Wilson, Andrew Gordon and Goldblum, Micah},
  year = {2025},
  month = jul,
  number = {arXiv:2507.07101},
  eprint = {2507.07101},
  primaryclass = {cs},
  doi = {10.48550/arXiv.2507.07101},
  urldate = {2025-08-05},
}

@inproceedings{Cai_2024_CVPR,
  title = {Batch Normalization Alleviates the Spectral Bias in Coordinate Networks},
  booktitle = {Proceedings of the {{IEEE}}/{{CVF}} Conference on Computer Vision and Pattern Recognition ({{CVPR}})},
  author = {Cai, Zhicheng and Zhu, Hao and Shen, Qiu and Wang, Xinran and Cao, Xun},
  year = {2024},
  month = jun,
  pages = {25160--25171},
}


```
